From a6f37b503bf2a50c7d708d2d25ebd6a1746898bb Mon Sep 17 00:00:00 2001
From: Dmitry Timoshkov <dmitry@baikal.ru>
Date: Mon, 29 Dec 2025 09:51:50 +0300
Subject: [PATCH] win32u: Also print hwnd in set_window_text() trace.

This makes +win log slightly more informative.

Signed-off-by: Dmitry Timoshkov <dmitry@baikal.ru>
(cherry picked from commit 5757d3a9a1c9ad0484fccaec2c8c62653d89882c)
---
 dlls/win32u/defwnd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dlls/win32u/defwnd.c b/dlls/win32u/defwnd.c
index d3067992aa0..65e1bf1ae4d 100644
--- a/dlls/win32u/defwnd.c
+++ b/dlls/win32u/defwnd.c
@@ -321,7 +321,7 @@ static BOOL set_window_text( HWND hwnd, const void *text, BOOL ansi )
     }
     else str = NULL;
 
-    TRACE( "%s\n", debugstr_w(str) );
+    TRACE( "%p, %s\n", hwnd, debugstr_w(str) );
 
     if (!(win = get_win_ptr( hwnd )))
     {
